source/mus_flow_module.fpp

```diff
@@ -140,6 +140,7 @@ contains
       ! lattice boltzmann passive scalar
       call mus_init_advRel_lbm_ps( relaxation = scheme%header%relaxation, &
         &                          layout     = scheme%header%layout, &
+        &                          order      = scheme%header%order, &
```

Comment:
To be replaced with `relaxation_variant = scheme%header%relaxation_options%variant`